### PR TITLE
Feature/valid trapi 1.1 response on error

### PR DESCRIPTION
**Pull request by xu-hao**

### CODE REVIEW

File: icees_api/features/knowledgegraph.py [hunk 1]
```diff
@@ -607,6 +607,7 @@ def p_values(feature_list):
             "datetime": datetime.datetime.now().strftime("%Y-%m-%D %H:%M:%S"),
             "message_code": "Error",
             "code_description": traceback.format_exc(),
+            "query_graph": query_graph
```
Comment (patrickkwang) [inline icees_api/features/knowledgegraph.py:610]:
```suggestion
            "query_graph": query_graph,
```

File: test/api/test_knowledge_graph.py [hunk 2]
```diff
@@ -139,7 +139,120 @@ def test_knowledge_graph_one_hop(query_options):
...
+    6,2010,0-2,>1,2
+    7,2010,0-2,0,3
+    8,2010,0-2,1,3
+    9,2010,0-2,>1,3
+    10,2010,0-2,0,4
+    11,2010,0-2,1,4
+    12,2010,0-2,>1,4
+""")
+def test_knowledge_graph_one_hop_valid_trapi_1_1_response(query_options):
+    """Test one-hop return valid TRAPI 1.1 response."""
```
Comment (patrickkwang) [inline test/api/test_knowledge_graph.py:160]: We could use [reasoner-validator](https://github.com/NCATSTranslator/reasoner-validator) for this purpose.

**Review by patrickkwang — CHANGES_REQUESTED** [review]

File: test/api/test_knowledge_graph.py (hunk 2 above)
Comment (xu-hao) [reply test/api/test_knowledge_graph.py:160]: just created a new branch to do that.

**Comment by xu-hao** [comment]
subsumed by #129

**Pull request closed without merging**